ansible_collections/arista/avd/molecule/eos_cli_config_gen/inventory/host_vars/router-bgp-base.yml

```diff
@@ -17,3 +17,11 @@ router_bgp:
       attribute_map: RM-ATTRIBUTE
       match_map: RM-MATCH
       advertise_only: true
+  address_family_ipv4:
+    networks:
+      10.0.0.0/8:
+      172.16.0.0/12:
+      192.168.0.0/16:
+  address_family_ipv6:
+    networks:
+      2001:db8::/32:
```

Comment:
please add another network where you test the route-map.

Reply:
Sure - done